dkist/net/attrs.py

```diff
@@ -20,6 +20,30 @@
 
 # SimpleAttrs
 
+class PageSize(_SimpleAttr):
+    """
+    The number of search results to return.
+
+    Parameters
+    ----------
+    page_size: `int`
+    """
+    def __init__(self, page_size: int):
+        super().__init__(page_size)
+
+
+class Page(_SimpleAttr):
```

Comment:
I went with page because the number seemed redundant "Page 2" reads better to me than "Page Number 2". I haven't really kept parity with the dataset search parameter names at all (in fact gone out of my way to hide them lol).

Reply:
Ah, I see. That makes a lot of sense.